catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/validation/impl/DecimalRangeValidatorTest.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.validation.impl;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.validation.impl.validator.DecimalRangeValidator;
+import ddf.catalog.validation.report.AttributeValidationReport;
+
+public class DecimalRangeValidatorTest {
```

Comment:
These classes would be excellent candidates for parametrized tests (see http://junit.sourceforge.net/javadoc/org/junit/runners/Parameterized.html).